### DISCUSSION TITLE
Redefining the I2S pins

### DISCUSSION THREAD

**Discussion by technoblogy** [pschatzmann/arduino-audio-tools#802] [Q&A] [answered]
I'm using arduino-audio-tools with an I2S amplifier board I've designed. What's the preferred way to redefine the I2S pin assignments? I can get my board to work if I edit the definitions in AudioConfig.h, such as:

    #define PIN_I2S_BCK 12
    #define PIN_I2S_WS 13
    #define PIN_I2S_DATA_OUT 14

However, I'd rather have a solution that allows me to edit the pin assignments in my source file. Sorry, but I can't find any documentation about this.

**Comment by pschatzmann** [comment, marked as answer]
3 ways:
- update AudioConfig.h directly
- create a AudioConfigLocal.h
- change it in the config object in the sketch: see https://github.com/pschatzmann/arduino-audio-tools/wiki/External-DAC

**Comment by technoblogy** [comment]
Thank you.
